python/lsst/pipe/analysis/colorAnalysis.py

```diff
@@ -47,13 +52,15 @@ class ColorTransform(Config):
                               doc="Intercept for lower fit line limits")
 
     @classmethod
-    def fromValues(cls, description, subDescription, plot, coeffs, requireGreater={}, requireLess={},
-                   fitLineSlope=None, fitLineUpperIncpt=None, fitLineLowerIncpt=None):
+    def fromValues(cls, description, subDescription, plot, coeffs, x0=None, y0=None, requireGreater={},
+                   requireLess={}, fitLineSlope=None, fitLineUpperIncpt=None, fitLineLowerIncpt=None):
```

Comment:
Using empty dicts as default arguments is dangerous. I think you haven't seen any weird side effects because none of the downstream code modifies them? You're defining Configs, so I'd expect this to be the case, but I doubt they're being used as configs and being frozen (since you didn't notice that defaults can't be None).  The usual way of defining empty dicts as function defaults. 
```
def fromValues(requireLess=None):
    if requireLess is None:
        requireLess = {}
```

Reply:
Fixed and made fields optional so they could default to `None` (see above).